cmdline-pull/pubsub_sample.py

```diff
@@ -104,80 +74,73 @@ def list_topics(client, args):
 
 
 def list_subscriptions(client, args):
-    """Shows the list of current subscriptions."""
-    next_page_token = None
-    while True:
-        resp = client.projects().subscriptions().list(
-            project='projects/{}'.format(args[0]),
-            pageToken=next_page_token).execute(num_retries=NUM_RETRIES)
-        for subscription in resp['subscriptions']:
-            print json.dumps(subscription, indent=1)
-        next_page_token = resp.get('nextPageToken')
-        if not next_page_token:
-            break
+    """Show the list of current subscriptions attached to a given topic.
```

Comment:
I feel the summary is not correct. How about to remove "attached to a given topic" and tweak the following details accordingly?